src/interface/others/HolyPriestSpreadsheet.js

```diff
@@ -145,23 +140,8 @@ const HolyPriestSpreadsheet = props => {
   };
 
   const healingFromAzerite = () => {
-    let totalHealing = 0;
+    const totalHealing = 0;
 
-    if (parser.getModule(BlessedSanctuary)) {
-      totalHealing += parser.getModule(BlessedSanctuary).effectiveHealing;
-    }
-    if (parser.getModule(EverlastingLight)) {
-      totalHealing += parser.getModule(EverlastingLight).effectiveHealing;
-    }
-    if (parser.getModule(PermeatingGlow)) {
-      totalHealing += parser.getModule(PermeatingGlow).permiatingGlowTotalHealAmount;
-    }
-    if (parser.getModule(PrayerfulLitany)) {
-      totalHealing += parser.getModule(PrayerfulLitany).rawPrayerfulLitanyHealing;
-    }
-    if (parser.getModule(WordOfMending)) {
-      totalHealing += parser.getModule(WordOfMending).effectiveHealing;
-    }
     return totalHealing;
```

Comment:
Probably better to just remove this function entirely, and then where it's used (just later on in this same file) replace the function call with 0